extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/Trigger.java

```diff
@@ -28,4 +28,10 @@
      */
     Instant getPreviousFireTime();
 
+    /**
+     *
+     * @return true if the last execution of the trigger is overdue.
```

Comment:
It would not hurt to mention that the period is configurable. WDYT?

Reply:
I changed it for: @return true if the last execution of the trigger is overdue. The grace period is configurable 

let me know if it's enough !

Reply:
It depends ;-). If we move the config to the `@Scheduled#overduePeriod()` then a simple link would be enough I guess.